main/configuration.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+
+	"gopkg.in/yaml.v2"
+)
+
+//const defaultCfgPATH = ""/etc/exquisite/config.yml""
+const defaultCfgPATH = "config.yml"
+
+// Configuration file structure
+type Configuration struct {
+	Debug      bool   `yaml:"debug"`
+	ServerPort int    `yaml:"server_port"`
+	PgDbURL    string `yaml:"pg_db_url"`
+	PgDatabase string `yaml:"pg_database"`
+	PgUsername string `yaml:"pg_username"`
+	PgPassword string `yaml:"pg_password"`
+}
+
+// InitConfiguration method for writeConfiguration func
+func (c *Configuration) InitConfiguration(debug bool) Configuration {
```

Comment:
If used only in writeConfiguration, should be private

Reply:
ok, will do it

Reply:
ok